### ISSUE TITLE
[BUG] Can't use phoenix without timestamps

### ISSUE THREAD

**Issue by wish2023** [Arize-ai/phoenix#692]
**Describe the bug**
For my image classification tasks, I'm using embedding vectors of size 1024. I am able to launch the app, but cannot see anything once I click on the URL to open Phoenix in my browser

**To Reproduce**
1. Copy the computer vision notebook locally
2. Serve my images on a web server
3. Replace `prod_df` and `train_df` with my custom dataset, which doesn't have a timestamp column, and larger embeddings
4. Run notebook to launch the app and click on the provided URL to access the app in my web browser (Chrome)

**Expected behavior**
Phoenix UI to open as it did with the dataset provided in the tutorial

**Screenshots**
![image](https://user-images.githubusercontent.com/60290739/236813056-64736760-8c2f-4d01-99ef-b69902b39a09.png)
![image](https://user-images.githubusercontent.com/60290739/236813508-d3c6c675-78a6-4402-9b52-2f691f614f7c.png)

**Environment (please complete the following information):**
 - OS: [Ubuntu 20.04.6]
 - Notebook Runtime [Jupyter]
 - Browser [Chrome]
 - Version [0.0.18]

**Comment by mikeldking** [comment]
Hi @wish2023 thanks so much for your bug report! The embedding size shouldn't have been a problem from our perspective. Will try to reproduce in the coming week or so. If there is any additional context you can add, please let us know!  Even a small sample of what your data may look like could help us a ton. 

Few questions that could help me a bit:
- Looks like it may have worked with our examples? Just trying to eliminate a platform / OS level issue
- Any chance you have a working colab or notebook you could share? Even a small `df.head()` could help us.
- Are your embeddings uniform in size at 1024?

Thanks in advance!

**Comment by wish2023** [comment]
Hi @mikeldking thank you for the quick response! I seem to have got the UI to load by converting my embeddings to a numpy array from a list. However, when I click on "image_embedding" in the "Embeddings" section I encounter the following error, which I believe is because I don't have a timestamp column. Is there any workaround to access the UI if my data doesn't have any timestamps associated with it? I don't want to view the time series of the embedding drift but just view the clustering of all my data.

![image](https://user-images.githubusercontent.com/60290739/236985323-18d59886-9bf7-4f4d-aaab-1faf0a26c835.png)

I have also included the head of my dataframe for your reference

![image](https://user-images.githubusercontent.com/60290739/236985451-80412efe-7df7-4a09-9e25-154b81bd5906.png)

**Comment by mikeldking** [comment]
Was able to reproduce the error explicitly on Ubunto (strangely it does not occur on colab which I would assume is a flavor of linux close to Ubuntu)

stack trace
```
Cannot compare tz-naive and tz-aware datetime-like objects.

GraphQL request:11:7
11 |       euclideanDistanceTimeSeries: driftTimeSeries(metric: euclideanDistance, ti
   |       ^
   | meRange: $timeRange, granularity: $driftGranularity) {
Traceback (most recent call last):
  File "/home/parallels/.local/lib/python3.10/site-packages/graphql/execution/execute.py", line 521, in execute_field
    result = resolve_fn(source, info, **args)
  File "/home/parallels/.local/lib/python3.10/site-packages/strawberry/schema/schema_converter.py", line 519, in _resolver
    return _get_result(_source, strawberry_info, **kwargs)
  File "/home/parallels/.local/lib/python3.10/site-packages/strawberry/schema/schema_converter.py", line 511, in _get_result
    return field.get_result(
  File "/home/parallels/.local/lib/python3.10/site-packages/strawberry/field.py", line 169, in get_result
    return self.base_resolver(*args, **kwargs)
  File "/home/parallels/.local/lib/python3.10/site-packages/strawberry/types/fields/resolver.py", line 186, in __call__
    return self.wrapped_func(*args, **kwargs)
  File "/home/parallels/.local/lib/python3.10/site-packages/phoenix/server/api/types/EmbeddingDimension.py", line 134, in drift_time_series
    data=get_drift_timeseries_data(self.dimension, metric, time_range, granularity)
  File "/home/parallels/.local/lib/python3.10/site-packages/phoenix/server/api/types/TimeSeries.py", line 123, in get_drift_timeseries_data
    return _get_timeseries_data(dimension, metric, time_range, granularity)
  File "/home/parallels/.local/lib/python3.10/site-packages/phoenix/server/api/types/TimeSeries.py", line 79, in _get_timeseries_data
    return df.pipe(
  File "/home/parallels/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5926, in pipe
    return common.pipe(self, func, *args, **kwargs)
  File "/home/parallels/.local/lib/python3.10/site-packages/pandas/core/common.py", line 518, in pipe
    return func(obj, *args, **kwargs)
  File "/home/parallels/.local/lib/python3.10/site-packages/phoenix/metrics/timeseries.py", line 88, in _aggregator
    return pd.concat(
  File "/home/parallels/.local/lib/python3.10/site-packages/pandas/core/reshape/concat.py", line 372, in concat
    op = _Concatenator(
  File "/home/parallels/.local/lib/python3.10/site-packages/pandas/core/reshape/concat.py", line 426, in __init__
    objs = list(objs)
  File "/home/parallels/.local/lib/python3.10/site-packages/phoenix/metrics/timeseries.py", line 198, in _results
    yield filtered.groupby(
  File "/home/parallels/.local/lib/python3.10/site-packages/pandas/core/indexing.py", line 1097, in __getitem__
    return self._getitem_tuple(key)
  File "/home/parallels/.local/lib/python3.10/site-packages/pandas/core/indexing.py", line 1289, in _getitem_tuple
    return self._getitem_tuple_same_dim(tup)
  File "/home/parallels/.local/lib/python3.10/site-packages/pandas/core/indexing.py", line 955, in _getitem_tuple_same_dim
    retval = getattr(retval, self.name)._getitem_axis(key, axis=i)
  File "/home/parallels/.local/lib/python3.10/site-packages/pandas/core/indexing.py", line 1323, in _getitem_axis
    return self._get_slice_axis(key, axis=axis)
  File "/home/parallels/.local/lib/python3.10/site-packages/pandas/core/indexing.py", line 1355, in _get_slice_axis
    indexer = labels.slice_indexer(slice_obj.start, slice_obj.stop, slice_obj.step)
  File "/home/parallels/.local/lib/python3.10/site-packages/pandas/core/indexes/datetimes.py", line 636, in slice_indexer
    return Index.slice_indexer(self, start, end, step)
  File "/home/parallels/.local/lib/python3.10/site-packages/pandas/core/indexes/base.py", line 6341, in slice_indexer
    start_slice, end_slice = self.slice_locs(start, end, step=step)
  File "/home/parallels/.local/lib/python3.10/site-packages/pandas/core/indexes/base.py", line 6534, in slice_locs
    start_slice = self.get_slice_bound(start, "left")
  File "/home/parallels/.local/lib/python3.10/site-packages/pandas/core/indexes/base.py", line 6449, in get_slice_bound
    label = self._maybe_cast_slice_bound(label, side)
  File "/home/parallels/.local/lib/python3.10/site-packages/pandas/core/indexes/datetimes.py", line 597, in _maybe_cast_slice_bound
    self._data._assert_tzawareness_compat(label)
  File "/home/parallels/.local/lib/python3.10/site-packages/pandas/core/arrays/datetimes.py", line 744, in _assert_tzawareness_compat
    raise TypeError(
TypeError: Cannot compare tz-naive and tz-aware datetime-like objects.
Stack (most recent call last):
  File "/usr/lib/python3.10/threading.py", line 973, in _bootstrap
    self._bootstrap_inner()
  File "/usr/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.10/threading.py", line 953, in run
    self._target(*self._args, **self._kwargs)
  File "/home/parallels/.local/lib/python3.10/site-packages/uvicorn/server.py", line 61, in run
    return asyncio.run(self.serve(sockets=sockets))
  File "/usr/lib/python3.10/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/usr/lib/python3.10/asyncio/base_events.py", line 633, in run_until_complete
    self.run_forever()
  File "/usr/lib/python3.10/asyncio/base_events.py", line 600, in run_forever
    self._run_once()
  File "/usr/lib/python3.10/asyncio/base_events.py", line 1896, in _run_once
    handle._run()
  File "/usr/lib/python3.10/asyncio/events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "/home/parallels/.local/lib/python3.10/site-packages/starlette/middleware/base.py", line 70, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "/home/parallels/.local/lib/python3.10/site-packages/starlette/middleware/exceptions.py", line 68, in __call__
    await self.app(scope, receive, sender)
  File "/home/parallels/.local/lib/python3.10/site-packages/starlette/routing.py", line 718, in __call__
    await route.handle(scope, receive, send)
  File "/home/parallels/.local/lib/python3.10/site-packages/starlette/routing.py", line 276, in handle
    await self.app(scope, receive, send)
  File "/home/parallels/.local/lib/python3.10/site-packages/strawberry/asgi/__init__.py", line 48, in __call__
    await self.http_handler_class(
  File "/home/parallels/.local/lib/python3.10/site-packages/strawberry/asgi/handlers/http_handler.py", line 50, in handle
    response = await self.get_http_response(
  File "/home/parallels/.local/lib/python3.10/site-packages/strawberry/asgi/handlers/http_handler.py", line 148, in get_http_response
    result = await execute(
  File "/home/parallels/.local/lib/python3.10/site-packages/strawberry/asgi/handlers/http_handler.py", line 201, in execute
    return await self.schema.execute(
  File "/home/parallels/.local/lib/python3.10/site-packages/strawberry/schema/schema.py", line 226, in execute
    result = await execute(
  File "/home/parallels/.local/lib/python3.10/site-packages/strawberry/schema/execute.py", line 144, in execute
    process_errors(result.errors, execution_context)
Cannot compare tz-naive and tz-aware datetime-like objects.

GraphQL request:11:7
11 |       euclideanDistanceTimeSeries: driftTimeSeries(metric: euclideanDistance, ti
   |       ^
   | meRange: $timeRange, granularity: $driftGranularity) {
Traceback (most recent call last):
  File "/home/parallels/.local/lib/python3.10/site-packages/graphql/execution/execute.py", line 521, in execute_field
    result = resolve_fn(source, info, **args)
  File "/home/parallels/.local/lib/python3.10/site-packages/strawberry/schema/schema_converter.py", line 519, in _resolver
    return _get_result(_source, strawberry_info, **kwargs)
  File "/home/parallels/.local/lib/python3.10/site-packages/strawberry/schema/schema_converter.py", line 511, in _get_result
    return field.get_result(
  File "/home/parallels/.local/lib/python3.10/site-packages/strawberry/field.py", line 169, in get_result
    return self.base_resolver(*args, **kwargs)
  File "/home/parallels/.local/lib/python3.10/site-packages/strawberry/types/fields/resolver.py", line 186, in __call__
    return self.wrapped_func(*args, **kwargs)
  File "/home/parallels/.local/lib/python3.10/site-packages/phoenix/server/api/types/EmbeddingDimension.py", line 134, in drift_time_series
    data=get_drift_timeseries_data(self.dimension, metric, time_range, granularity)
  File "/home/parallels/.local/lib/python3.10/site-packages/phoenix/server/api/types/TimeSeries.py", line 123, in get_drift_timeseries_data
    return _get_timeseries_data(dimension, metric, time_range, granularity)
  File "/home/parallels/.local/lib/python3.10/site-packages/phoenix/server/api/types/TimeSeries.py", line 79, in _get_timeseries_data
    return df.pipe(
  File "/home/parallels/.local/lib/python3.10/site-packages/pandas/core/generic.py", line 5926, in pipe
    return common.pipe(self, func, *args, **kwargs)
  File "/home/parallels/.local/lib/python3.10/site-packages/pandas/core/common.py", line 518, in pipe
    return func(obj, *args, **kwargs)
  File "/home/parallels/.local/lib/python3.10/site-packages/phoenix/metrics/timeseries.py", line 88, in _aggregator
    return pd.concat(
  File "/home/parallels/.local/lib/python3.10/site-packages/pandas/core/reshape/concat.py", line 372, in concat
    op = _Concatenator(
  File "/home/parallels/.local/lib/python3.10/site-packages/pandas/core/reshape/concat.py", line 426, in __init__
    objs = list(objs)
  File "/home/parallels/.local/lib/python3.10/site-packages/phoenix/metrics/timeseries.py", line 198, in _results
    yield filtered.groupby(
  File "/home/parallels/.local/lib/python3.10/site-packages/pandas/core/indexing.py", line 1097, in __getitem__
    return self._getitem_tuple(key)
  File "/home/parallels/.local/lib/python3.10/site-packages/pandas/core/indexing.py", line 1289, in _getitem_tuple
    return self._getitem_tuple_same_dim(tup)
  File "/home/parallels/.local/lib/python3.10/site-packages/pandas/core/indexing.py", line 955, in _getitem_tuple_same_dim
    retval = getattr(retval, self.name)._getitem_axis(key, axis=i)
  File "/home/parallels/.local/lib/python3.10/site-packages/pandas/core/indexing.py", line 1323, in _getitem_axis
    return self._get_slice_axis(key, axis=axis)
  File "/home/parallels/.local/lib/python3.10/site-packages/pandas/core/indexing.py", line 1355, in _get_slice_axis
    indexer = labels.slice_indexer(slice_obj.start, slice_obj.stop, slice_obj.step)
  File "/home/parallels/.local/lib/python3.10/site-packages/pandas/core/indexes/datetimes.py", line 636, in slice_indexer
    return Index.slice_indexer(self, start, end, step)
  File "/home/parallels/.local/lib/python3.10/site-packages/pandas/core/indexes/base.py", line 6341, in slice_indexer
    start_slice, end_slice = self.slice_locs(start, end, step=step)
  File "/home/parallels/.local/lib/python3.10/site-packages/pandas/core/indexes/base.py", line 6534, in slice_locs
    start_slice = self.get_slice_bound(start, "left")
  File "/home/parallels/.local/lib/python3.10/site-packages/pandas/core/indexes/base.py", line 6449, in get_slice_bound
    label = self._maybe_cast_slice_bound(label, side)
  File "/home/parallels/.local/lib/python3.10/site-packages/pandas/core/indexes/datetimes.py", line 597, in _maybe_cast_slice_bound
    self._data._assert_tzawareness_compat(label)
  File "/home/parallels/.local/lib/python3.10/site-packages/pandas/core/arrays/datetimes.py", line 744, in _assert_tzawareness_compat
    raise TypeError(
TypeError: Cannot compare tz-naive and tz-aware datetime-like objects.
Stack (most recent call last):
  File "/usr/lib/python3.10/threading.py", line 973, in _bootstrap
    self._bootstrap_inner()
  File "/usr/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.10/threading.py", line 953, in run
    self._target(*self._args, **self._kwargs)
  File "/home/parallels/.local/lib/python3.10/site-packages/uvicorn/server.py", line 61, in run
    return asyncio.run(self.serve(sockets=sockets))
  File "/usr/lib/python3.10/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/usr/lib/python3.10/asyncio/base_events.py", line 633, in run_until_complete
    self.run_forever()
  File "/usr/lib/python3.10/asyncio/base_events.py", line 600, in run_forever
    self._run_once()
  File "/usr/lib/python3.10/asyncio/base_events.py", line 1896, in _run_once
    handle._run()
  File "/usr/lib/python3.10/asyncio/events.py", line 80, in _run
    self._context.run(self._callback, *self._args)
  File "/home/parallels/.local/lib/python3.10/site-packages/starlette/middleware/base.py", line 70, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "/home/parallels/.local/lib/python3.10/site-packages/starlette/middleware/exceptions.py", line 68, in __call__
    await self.app(scope, receive, sender)
  File "/home/parallels/.local/lib/python3.10/site-packages/starlette/routing.py", line 718, in __call__
    await route.handle(scope, receive, send)
  File "/home/parallels/.local/lib/python3.10/site-packages/starlette/routing.py", line 276, in handle
    await self.app(scope, receive, send)
  File "/home/parallels/.local/lib/python3.10/site-packages/strawberry/asgi/__init__.py", line 48, in __call__
    await self.http_handler_class(
  File "/home/parallels/.local/lib/python3.10/site-packages/strawberry/asgi/handlers/http_handler.py", line 50, in handle
    response = await self.get_http_response(
  File "/home/parallels/.local/lib/python3.10/site-packages/strawberry/asgi/handlers/http_handler.py", line 148, in get_http_response
    result = await execute(
  File "/home/parallels/.local/lib/python3.10/site-packages/strawberry/asgi/handlers/http_handler.py", line 201, in execute
    return await self.schema.execute(
  File "/home/parallels/.local/lib/python3.10/site-packages/strawberry/schema/schema.py", line 226, in execute
    result = await execute(
  File "/home/parallels/.local/lib/python3.10/site-packages/strawberry/schema/execute.py", line 144, in execute
    process_errors(result.errors, execution_context)
```

strangely the df.head of the generated timestamps are identical. Still digging in.

**Comment by mikeldking** [comment]
hey @wish2023 thanks for your patience. We believe it's fixed in `arize-phoenix==0.0.19` - there's a bit of a bug in the ubuntu `pandas.groupby` that we didn't catch. Thanks for your detailed report, it helped a ton in debugging and solving this hard to repro bug. 

Please let us know if you encounter any more issues!
